packages/core/src/Site/index.js

```diff
@@ -889,6 +919,28 @@ class Site {
     });
   }
 
+  /**
+   * Handles the reloading of ignore attributes
+   */
+  async handleIgnoreReload(oldIgnore) {
+    const assetsToRemove = _.difference(this.siteConfig.ignore, oldIgnore);
```

Comment:
:eyes: should this be the other way round?

Reply:
Hmm I think this is right as new items that are added to ignore will have to be removed?

Reply:
👍 my bad, was looking from perspective of `siteConfig."include"`